### PR TITLE
Make CI pass

### PR DESCRIPTION
**Pull request by jhamman**

### CODE REVIEW

File: intake_stac/tests/test_catalog.py [hunk 1]
```diff
@@ -120,6 +121,7 @@ def test_cat_from_item(stac_item_obj):
     assert 'B5' in cat
 
 
+@pytest.mark.xfail(reason='need to fix stack bands')
```
Comment (jhamman) [inline intake_stac/tests/test_catalog.py:124]: see https://github.com/intake/intake-stac/issues/55

File: intake_stac/tests/test_catalog.py [hunk 2]
```diff
@@ -88,6 +88,7 @@ def test_init_catalog_with_bad_url_raises():
         StacCatalog('foo.bar')
 
 
+@pytest.mark.xfail(reason='need to fix serialization')
```
Comment (jhamman) [inline intake_stac/tests/test_catalog.py:91]: I think this can be fixed along with #31

**Comment by jhamman** [comment]
@scottyhq - will any of this conflict with what you've been doing?

**Comment by scottyhq** [comment]
> @scottyhq - will any of this conflict with what you've been doing?

feel free to merge and I will sort out any conflicts. still settling on 1.0 catalogs to use for examples and tests

**Pull request merged**